src/core/src/bootstrap/Constants.py

```diff
@@ -323,3 +323,9 @@ class EnvLayer(EnumBackport):
         NOT_SELECTED: 6
     }
 
+    # Ubuntu Pro Client constants.
+    class UbuntuProClientSettings(EnumBackport):
+        FEATURE_ENABLED = True,
```

Comment:
This flag is always true. This flag is used only in one if condition and in one log. Why do we need this flag?

Reply:
This is by design. In case if there is an issue that we discovered, we can quickly set this to FALSE and deploy, while we look into fixing the issue.